### PR TITLE
Add Vagrantfile and provision.sh

### LINKED ISSUE #319: There should be a Vagrant file for the project

### PR DESCRIPTION
**Pull request by lizconlan**
Adds a Vagrant file and a separate provision script and updates the README file.

Throws up a few depreciation and security warnings while it's installing packages, not sure if those are anything to worry about.

Closes #319

### CODE REVIEW

File: Vagrantfile-example [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+The website code is found in: /vagrant/yournextmp-popit
+
+You can run the dev server with: python manage.py runserver 0.0.0.0:8000"
+
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.network :forwarded_port, guest: 8000, host: YNMP_VAGRANT_PORT
+  config.vm.synced_folder ".", "/home/vagrant/yournextmp", :owner => "vagrant", :group => "vagrant"
```
Comment (andylolz) [inline Vagrantfile-example]: I wonder if this should instead say:

```
  config.vm.synced_folder ".", "/vagrant", :owner => "vagrant", :group => "vagrant"
```

**Comment by lizconlan** [comment]
@andylolz - does this look better now? I've included the changes from your PR here

**Comment by andylolz** [comment]
Yes thanks – much better!

**Comment by lizconlan** [comment]
\o/

**Pull request merged**